app/api/api_v1/schemas/document.py

```diff
@@ -177,6 +177,7 @@ class BulkImportValidatedResult(BaseModel):
 
     document_count: int
     document_added_count: int
+    document_updated_count: int
     document_skipped_count: int
```

Comment:
Suggesting changing the name "document_skipped_count". 

This is as we may see the following which could be confusing: 
document_count: 2 
document_added_count: 0 
document_updated_count: 1
document_skipped_count: 2

I would propose changing to the following or something similar:

document_count: 2 
document_ingest_count: 0 
document_existing_updated_count: 1
document_ingest_skipped_count: 2

Reply:
Even better `document_not_ingested_count` ? or even leave it off altogether if its `document_count` - `document_added_count`